src/components/stacked-area-chart/StackedAreaChart.tsx

```diff
@@ -156,88 +161,143 @@ function maxLengthOfAllArrays(series: Series[]): number {
-                            y={marY + 5 + chartY + (i * - 20)}
-                            // fill={series[i].color || colors[i]}
-                            className="chart-label"
-                        >
-                        {series[i].label}
-                        </text>
-                    )
-                }
-            </svg>  
-        </div>
-    );
-};
+                        {
+                        scaleYAxis(scale, biggest)
+                        }
+                    {/*Y axis*/}
+                    <line 
+                        x1={marX}
+                        x2={marX} 
+                        y1={marY} 
+                        y2={marY + chartY}
+                        stroke="black"
+                        strokeWidth="4" 
+                    />
+                    {/*X axis*/}
+                    <line 
+                        x1={marX - 2} 
+                        x2={marX + chartX - 0.5}
+                        y1={marY + chartY}
+                        y2={marY + chartY}
+                        stroke="black"
+                        strokeWidth="4" 
+                    />
+                    {/*Scale of X axis*/}
+                        {
+                            scaleXAxis(scale, maxLen, scaleLabel)
+                        }
+                    {/*Legend*/}
+                    {stackSeries.map((num, i) =>
+                            <circle
+                                key={i}
+                                cx={marX + chartX + 20}
+                                cy={marY + chartY + (i * - 20)}
+                                r={5}
+                                fill={stackSeries[i].color || colors[i]}
+                                opacity="1"
+                            />
+                        )
+                    }
+                    {stackSeries.map((num, i) =>
+                            <text
+                                key={i}
+                                x={marX + chartX + 30}
+                                y={marY + 5 + chartY + (i * - 20)}
+                                // fill={series[i].color || colors[i]}
+                                className="chart-label"
+                            >
+                            {stackSeries[i].label}
+                            </text>
+                        )
+                    }
+                    {this.drawRects(maxLen)}
+                </svg>
+                
+                <ReactTooltip
+                    place="top"
+                    type="light"
+                    effect="float"
+                    id="stacked-area-chart-tooltip"
+                    className="chart-tooltip"
+                >
+                    {hoverId !== -1 ? (
+                        <div>
+                            {series.map((num, i) =>
+                            <div key={i}>{series[i].label + ': ' + series[i].data[hoverId]}</div>
+                            )
```

Comment:
Just minor nitpick, you can put `)}` on the same line and indent `<div>`.

Reply:
Okay, let me.